lifelines/plotting.py

```diff
@@ -291,7 +291,7 @@ def qq_plot(model, ax=None, **plot_kwargs):
 
     max_, min_ = quantiles[COL_EMP].max(), quantiles[COL_EMP].min()
 
-    quantiles.plot.scatter(COL_THEO, COL_EMP, c="none", edgecolor="k", lw=0.5, ax=ax)
+    quantiles.plot.scatter(COL_THEO, COL_EMP, c="none", edgecolor=scatter_color, lw=0.5, ax=ax)
     ax.plot([min_, max_], [min_, max_], c="k", ls=":", lw=1.0)
```

Comment:
should the color here be the same as `scatter_color`?

Reply:
I'd recommend no (I'm assuming you're referring to the `c="k"` argument for ax.plot()?), since that line of code is simply for the diagonal reference line in the QQ plot - for the use case of overlaying qq plots, having different colored diagonals wouldnt make a difference since they all would overlap anyway!

Also sorry this took so long, I'm terrible at noticing github notifications!